lib/Dialect/TTIR/Transforms/Passes.cpp

```diff
@@ -730,17 +747,21 @@ class TTIRLayoutFuncReturnRewriter
   LogicalResult matchAndRewrite(mlir::func::ReturnOp op,
                                 PatternRewriter &rewriter) const final {
     bool modified = false;
-    for (auto &operand : op->getOpOperands()) {
+    for (int64_t operandIndex = 0; operandIndex < op.getNumOperands();
+         ++operandIndex) {
+      auto &operand = op->getOpOperand(operandIndex);
```

Comment:
Btw, this could have been written:
```c++
for (auto &operand : op->getOpOperands()) {
  unsigned operandIndex = operand.getOperandNumber();
```

[getOperandNumber](https://mlir.llvm.org/doxygen/classmlir_1_1OpOperand.html#a097f9026defd8afd19ab06b21aa11bdf)